### ISSUE TITLE
Support specifying individual image width and height after zoom-in via data attributes

### ISSUE THREAD

**Issue by kingdido999** [kingdido999/zooming#47]
We have a global configuration to manipulate image width and height after zoom-in via `scaleBase`, `scaleExtra` and `customSize`, but we lack the ability to customize each individual image.

I propose a feature to zoom image into the given width and height via data attributes:

```html
<img data-src="image.jpg" data-zooming-width="1920" data-zooming-height="1080" data-action="zooming" />
```

And they should overwrite any global image scaling configuration if applicable.

**Comment by kingdido999** [comment]
This feature is available in version `1.3.0`. [Documentation](https://desmonding.me/zooming/docs/#/guide?id=specify-image-width-and-height).

**Comment by allaire** [comment]
@kingdido999 This feature works well, but is there a way to do something like #65 and have something like "zoom at most to `data-zooming-width` and `data-zooming-height`, but if it's bigger than the viewport, zoom to viewport (and take into account `baseScale`)?
